src/Pages/Main/MainComponents/BigCardContents.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+function BigCardContents(props) {
```

Comment:
```suggestion
function BigCardContents({ title }) {
```
파라미터 자리에서 곧바로 구조분해 할당 가능합니다